code/modules/projectiles/guns/misc/beam_rifle.dm

```diff
@@ -143,6 +148,10 @@
 		return FALSE
 	animate(user.client, pixel_x = 0, pixel_y = 0, 0, FALSE, LINEAR_EASING, ANIMATION_END_NOW)
 	zoom_current_view_increase = 0
+	if iscarbon(user)
+		var/mob/living/carbon/human/H = user

```

Comment:
```suggestion
		var/mob/living/carbon/H = user
```
if youre curious why this is (probably) important, monkeys would pass the iscarbon check but wouldnt match this type